src/main/java/uk/gov/ida/verifyserviceprovider/services/NonMatchingAssertionService.java

```diff
@@ -72,7 +82,21 @@ public TranslatedNonMatchingResponseBody translateSuccessResponse(List<Assertion
 
         validate(authnAssertion, mdsAssertion, expectedInResponseTo, expectedLevelOfAssurance, levelOfAssurance);
 
+        validate(authnAssertion, mdsAssertion, expectedInResponseTo, expectedLevelOfAssurance, levelOfAssurance);
```

Comment:
I'm not sure how this has happened but we now seem to be calling the validate method twice in a row! :)